File: app/src/main/java/com/moldedbits/genesis/utils/Utilities.java
package com.moldedbits.genesis.utils;

import android.content.res.Resources;
import android.util.TypedValue;

public class Utilities {

    public static final String CATEGORY_KEY = "categoryKey";
    public static final String CATEGORY_NAME = "categoryName";
    public static final String PASSAGE_INDEX = "passageIndex";
    public static final String FB_PASSAGE_DETAILS = "passageDetails";

    public static float convertDpToPx(int pixel, Resources res) {
        return TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, pixel,
                res.getDisplayMetrics());
    }
}

File: languagetools/src/main/java/com/moldedbits/languagetools/Utilities.kt
package com.moldedbits.languagetools

import android.content.res.Resources
import android.util.TypedValue

object Utilities {

    val ARTICLE_ID = "articleId"

    fun convertDpToPx(pixel: Int, res: Resources): Float {
        return TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, pixel.toFloat(),
                res.displayMetrics)
    }
}